src/app/models/propTypes/fragment/index.js

```diff
@@ -0,0 +1,9 @@
+import { string, arrayOf, oneOf } from 'prop-types';
+import { blockOfSpecificTypeAndModel } from '../general';
+
+const fragmentBlockPropTypes = blockOfSpecificTypeAndModel('fragment', {
+  text: string.isRequired,
+  attributes: arrayOf(oneOf(['bold', 'italic'])).isRequired,
```

Comment:
Does this propType declaration also allow for an empty `attributes` array?

Reply:
Spoke with @jtart offline and this is not an issue as if attributes is an empty array, Fragments will just return the text ([details](https://github.com/BBC-News/simorgh/blob/7ecd927859c7eac756b14376358ef2bc13656722/src/app/containers/Fragment/index.jsx#L18)). This is because `text` is the initial value [here](https://github.com/BBC-News/simorgh/blob/7ecd927859c7eac756b14376358ef2bc13656722/src/app/containers/Fragment/index.jsx#L24) and the reduce on an empty array effectively returns nothing